<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Report</class>
 <widget class="QWidget" name="Report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="8">
    <widget class="QTableView" name="twReport"/>
   </item>
   <item row="0" column="7" rowspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Від</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QDateTimeEdit" name="sbStart"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Центрифуга №</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QComboBox" name="cf_no">
     <item>
      <property name="text">
       <string>Всі</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>До</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Відобразити за</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QDateTimeEdit" name="sbStop"/>
   </item>
   <item row="1" column="5" colspan="2">
    <widget class="QComboBox" name="cbSelZm">
     <item>
      <property name="text">
       <string>Поточна зміна</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Попердня зміна</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Поточна+Попередея</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Інший діапазон</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="bnStart">
     <property name="icon">
      <iconset>
       <normaloff>:/butons/pict/calendar_48x48.png</normaloff>:/butons/pict/calendar_48x48.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="bnStop">
     <property name="icon">
      <iconset>
       <normaloff>:/butons/pict/calendar_48x48.png</normaloff>:/butons/pict/calendar_48x48.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
